astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 3e044eb26395c8569e3e15bf821c6ae5eeea324a
+Subproject commit 160b48587680e11f12b66a51641a2601b869d1fa
```

Comment:
This morning, and there were (I think) other upstream updates which did not give any similar trouble.